Formula/phpbrew.rb

```diff
@@ -0,0 +1,19 @@
+class Phpbrew < Formula
+  desc "Brew & manage PHP versions in pure PHP at HOME"
+  homepage "https://phpbrew.github.io/phpbrew"
+  url "https://github.com/phpbrew/phpbrew/releases/download/1.27.0/phpbrew.phar"
+  sha256 "0fdcda638851ef7e306f5046ff1f9de291443656a35f5150d84368c88aa7a41a"
+  license "MIT"
+
+  bottle :unneeded
+
+  def install
+    chmod "+x", "phpbrew.phar"
+    bin.install "phpbrew.phar" => "phpbrew"
```

Comment:
Doesn't `bin.install` set permissions correctly on its own?

Reply:
Not if it wasn't executable to begin with I think.